metadata-ingestion/src/datahub/ingestion/source/unity/hive_metastore_proxy.py

```diff
@@ -38,6 +38,21 @@
     "binary": ColumnTypeName.BINARY,
 }
 
+column_stat_property_map = {
```

Comment:
Can we reuse this in `proxy_profiling.py`? Also, idk if we have to replicate properties like this... that was just a consequence of copying all properties from unity catalog. What we really want is the stats to show up on the stats page, not the properties page, so I think this might be unnecessary.

Reply:
updated this to keep hive metastore profiling separate.